vignettes/metastasisPhyloVision.Rmd

```diff
@@ -0,0 +1,77 @@
+---
+title: "metastasis"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+devtools::load_all("/home/eecs/yanayrosen/VISION/") # change to just load VISION
+library(ape)
+library(reticulate)
+```
+
+## R Markdown
+
+This is an R Markdown document. Markdown is a simple formatting syntax for authoring HTML, PDF, and MS Word documents. For more details on using R Markdown see <http://rmarkdown.rstudio.com>.
+
+When you click the **Knit** button a document will be generated that includes both content as well as the output of any embedded R code chunks within the document. You can embed an R code chunk like this:
+
+## Creating a Phylo Vision Object with a Tree
+
+Vision objects now support dendrograms for visualization and analysis. To create the Phylo-VISION object, you need all of the same data, an expression matrix, and metadata and/or signatures, but you also include a tree object from the Ape package.
+```{r create}
+# Read the expression and meta data
+expr <- read_10x(expression = "/data/yosef2/users/mattjones/projects/PhyloVision/data/metastasis_data/GRCh38/matrix.mtx",
+                 genes = "/data/yosef2/users/mattjones/projects/PhyloVision/data/metastasis_data/GRCh38/genes.tsv",
+                 barcodes = "/data/yosef2/users/mattjones/projects/PhyloVision/data/metastasis_data/GRCh38/barcodes.tsv")
+
+
+barcodes <- read.table("/data/yosef2/users/mattjones/projects/PhyloVision/data/metastasis_data/GRCh38/barcodes.tsv", check.names = FALSE, sep="\t")
+latent <- read.table("/data/yosef2/users/mattjones/projects/PhyloVision/data/metastasis_data/latent.csv", check.names=FALSE, sep="\t", row.names = 1, header=TRUE)
+row.names(latent) <- barcodes[, 1]
+
+meta <- read.table("/data/yosef2/users/mattjones/projects/PhyloVision/data/metastasis_data/GSM4905335_meta.5k.tsv.gz", check.names = FALSE, sep = "\t", row.names = 1, header=TRUE)
+# meta$Cell_Type <- as.factor(meta$Cell_Type)
+# Signature file
+sigs = c("c5.go.bp.v7.2.symbols.gmt", "h.all.v5.2.symbols.gmt", "c4.cgn.v7.2.symbols.gmt")
+
+# Read the trees
+lg7_tree <- read.tree("/data/yosef2/users/mattjones/projects/PhyloVision/data/metastasis_data/lg7_tree_hybrid_priors.alleleThresh.processed.txt")
+lg4_tree <- read.tree("/data/yosef2/users/mattjones/projects/PhyloVision/data/metastasis_data/lg4_tree_hybrid_priors.alleleThresh.processed.txt")
+
```

Comment:
No need to do the analysis for both trees -- let's only use the LG4 tree.

Also, can we add this tree to the `data` directory so it's distributed with the package?

Reply:
For file locations should I just put like "..." or something else? How do we want to distribute the data?

Reply:
Updated to have users set a path variable-- still not sure how to distribute the data though?

Reply:
^ @mattjones315